### DISCUSSION TITLE
New feature to fight gap fill slowness : Gap fill only on top and bottom layers ? ironing (filled) ?

### DISCUSSION THREAD

**Discussion by DzzD** [supermerill/SuperSlicer#1865] [General]
Gap fill is often something that add a significant amount of time to my prints (between 10 to 20%).

Gap fill is a lot more interresting in bottom and top layers but not that much inside the model.

So dont you think an option "gap fill only bottom & top layer" would be a great new feature ?

But It is already nearly doable by disabling "Gap fill" and using "Monotonic (filled)" for infill but there is no such a "Ironing (filled)", may be only adding an option of "ironing (filled)" for infill top layer may do the job ?

**Comment by supermerill** [comment]
#1795
please use issue for feature request.
